File: pset6/mario/mario.py
from cs50 import get_int


height = get_int("Height: ")

while height>23 or height<0:
    height = get_int("Height: ")
for i in range(height):
    blankSpace = height - (i+1)
    for x in range(blankSpace):
        print(" ", end='')
    for x in range(i+2):
        print("#", end='')
    print();

File: pset1/mario/less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void){

    int height;

    do{
         height = get_int("Height: ");
    }
    while(height>23 || height<0);


    for(int i=0; i<height; i++){
        // for every ith row, print height-ith+1 black spaces
        int blankSpace = height-(i+1);
        for(int x=0; x<blankSpace; x++){
            printf(" ");
        }
        //for every ith row, print i+2 #
        for(int x=0; x<(i+2); x++){
            printf("#");
        }
        printf("\n");


    }



}